var/spack/repos/builtin/packages/symengine/package.py

```diff
@@ -38,9 +40,6 @@ class Symengine(CMakePackage):
             description='Enable thread safety option')
     variant('shared',       default=True,
             description='Enables the build of shared libraries')
-    variant('build_type', default='Release',
```

Comment:
`build_type` is a standard `variant` for `CmakePackage`, here it's overwritten to only support two options. I think `build_type=Release` is the right way. 

I am not sure, but I hope 
```
depends_on('symengine build_type=debug',...)
```
works in `dealii`.

Reply:
> build_type is a standard variant for CmakePackage

Ok, I saw the `debug` variant in Trilinos and thought that must be right. I guess its just a legacy option then. I'll revert this change.